### ISSUE TITLE
Average time in Compile Score Timeline

### ISSUE THREAD

**Issue by FrozenInMTL** [Viladoman/CompileScore#30]
It would be nice to have a option to show the Compile Score Timeline with the average time, if you have a lot of modules with different pch that would be really helpful.

**Comment by Viladoman** [comment]
I am not sure I follow on what would you need exactly. Please feel free to contact me directly with more details. I would love to know more.

**Comment by Viladoman** [comment]
I finally got some time to work on Compile Score again. This got added to Version 1.9.0 so that we get metrics not only of a given header but also in its relationship between includer and includee. 
Also a new table in the overview got added to find bottlenecks at the specific #include level. 

![image](https://github.com/Viladoman/CompileScore/assets/8118743/07002445-ce16-454f-8965-284fb76c8f54)

Thanks for the suggestion, this information is really super helpful and it leads faster to actionable points when getting new profile data.
